### ISSUE TITLE
UX improvements

### ISSUE THREAD

**Issue by evertonfraga** [ethereum/mist#1455]
This is a list to gather and discuss UX improvements on Mist, to be a friendly tool for the general public. Will be done as time allows. 

### Browser bar

- [ ] Focus on `F6` or `Cmd + L`
- [ ] `Cmd + T` to open a blank page

### Sidebar

- [ ] Select tabs via `Control or command + [1-9]` (#1454)
- [ ] Deal with large amount of tabs (scroll)
- [ ] Make sidebar and url bar consistently aligned among OSes

Please collaborate.

**Comment by ajw107** [comment]
Just for those of us without perfect eyesight, the option to choose:
- font colour (especially when fetching blocks on start up, as grey against grey is a bit mad)
- style (bold font for people with double vision or just generally poor viison can be a life saver)
- size (again some people can only see through a small field, others through a large field)
- face (some fonts can be too complex to make out, eg arial over times)
This would be an invaluable addition to a great app.

**Comment by evertonfraga** [comment]
Thanks for your input @ajw107. As Mist is getting more mature (and have more staff), we're starting to look more closely to those issues.

**Comment by timelinefunds** [comment]
Cools (y)

**Comment by MysticRyuujin** [comment]
I'd like to see "Minimize to System Tray" as well in Windows. I'd like to run it as a background application so having it hidden in the system tray as opposed to the taskbar would be great.

**Comment by luclu** [comment]
@MysticRyuujin there is already an issue for the tray functionality, here you go: https://github.com/ethereum/mist/issues/85

**Comment by jamesray1** [comment]
I agree with @MysticRyuujin. I also would like to see Mist start with system startup.

**Comment by jamesray1** [comment]
Update: it is fairly easy to make a program startup in Windows. This article [here](http://www.bnewtech.com/make-a-program-run-at-startup-in-windows/) outlines how to do it for Windows 8–10.

**Comment by agilob** [comment]
+1 for hiding to tray support

**Comment by ZyXoid** [comment]
+1 for hiding to tray for Windows
